deployability/modules/workflow_engine/examples/test/vagrant/test-agent-complete.yaml

```diff
@@ -0,0 +1,114 @@
+version: 0.1
+description: This workflow is used to test agents deployment por DDT1 PoC
```

Comment:
```suggestion
description: This workflow is used to test agents' deployment for DDT1 PoC
```

Reply:
Done in [6311d92](https://github.com/wazuh/wazuh-qa/pull/5190/commits/6311d92724994e505494d41ca00dd7d9cddda76f)